lib/src/components/NxNavigationSidebar/NxNavigationSidebar.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import NxButton from '../NxButton/NxButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxNavigationSidebar.scss';
+
+const NxNavigationSidebar: FunctionComponent<Props> = function NxNavigationSidebar(props) {
+  const {
+    isOpen,
+    children,
+    className,
+    toggleOpenIcon,
+    toggleCloseIcon,
+    onToggleClick,
+    logoImg,
+    logoAltText,
+    logoLink
+  } = props;
+
+  const classes = classnames(className, 'nx-page-sidebar-operable', {
+    'open': isOpen,
+    'closed': !isOpen
+  });
+
+  const toggleButtonIcon = isOpen ? toggleOpenIcon : toggleCloseIcon;
+  const toggleButton = (
+    <NxButton aria-label="toggle"
```

Comment:
Also consider setting `aria-expanded` and `aria-controls` as described here: https://www.w3.org/TR/wai-aria/#aria-expanded

Reply:
> Also consider setting `aria-expanded` and `aria-controls` as described here: https://www.w3.org/TR/wai-aria/#aria-expanded

Added in 60cba793dbc0a1c1708ad8a60766d855f13c941c